core/src/com/mygdx/crazybomber/model/player/Player.java

```diff
@@ -114,14 +114,22 @@ public Bomb dropBomb() {
         } else {
             droppedBomb.setXCoordinate((int) Math.round(getYCoordinate()));
         }
+        ScheduledExecutorService _scheduledExecutorService = Executors.newScheduledThreadPool(5);
+        System.out.println("bomb dropped");
+        ScheduledFuture _scheduledFuture =
+                _scheduledExecutorService.schedule(new Runnable() {
+                    public void run() {
+                        droppedBomb.explode();
+                    }
+                }, 3, TimeUnit.SECONDS);
+        _scheduledExecutorService.shutdown();
```

Comment:
Instead of shutting down maybe we can assign a thread to a bomb to be running until end of game.

Reply:
stop it